### ISSUE TITLE
It does not work in `Amazon Linux 2023`

### ISSUE THREAD

**Issue by OneideLuizSchneider** [ossec/ossec-hids#2090]
It does not work on `Amazon Linux 2023`. 
I did try to add manually some libs but not successfully. 
I downgraded it back to `Amazon Linux 2`.

Not sure if someone else had the same problem.

**Comment by bigtrucker89** [comment]
Amazon Linux 2023 is only supported with Atomic OSSEC at this time.

**Comment by sempervictus** [comment]
@atomicturtle - ABI problem or something else? Is this a "buildbots need to include it" concern?

**Comment by atomicturtle** [comment]
Yeah just builders. Does anyone know if amazon linux 2023 is a LTS version like amzn2?

**Comment by OneideLuizSchneider** [comment]
@atomicturtle `Amazon Linux 2023` is the default version for all new EC2 instances.

https://aws.amazon.com/linux/amazon-linux-2023/

**Comment by sempervictus** [comment]
Yes, it's an LTS

**Comment by atomicturtle** [comment]
Looks fine here, I used the same setup for Rocky 8:
dnf -y groupinstall "Development Tools"
dnf -y install pcre2-devel openssl-devel systemd-devel
<snip>


DLIBOPENSSL_ENABLED
Compiler:
    CFLAGS          -I./external/compat -DMAX_AGENTS=2048 -DOSSECHIDS -DDEFAULTDIR="/var/ossec" -DUSER="ossec" -DREMUSER="ossecr" -DGROUPGLOBAL="ossec" -DMAILUSER="ossecm" -DLinux -DINOTIFY_ENABLED -DHAVE_SYSTEMD -DZLIB_SYSTEM -DUSE_PCRE2_JIT -DLIBOPENSSL_ENABLED -Wall -Wextra -I./ -I./headers/
    LDFLAGS          -lm -lpthread -lsystemd -lpcre2-8 -lssl -lcrypto -lz
    CC              cc
    MAKE            make
make[1]: Leaving directory '/root/ossec-hids/src'

Done building server
[root@fbcd80e374d9 src]# cat /etc/system-release
Amazon Linux release 2023 (Amazon Linux)

Closing this one out
